### PR TITLE
Offshoot Search (Tiles): Un-bold "By" lines; remove when empty field

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-5289

### CODE REVIEW

**Review by dualcnhq — APPROVED** [review]

File: src/tiles/grid/item-tile.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class ItemTile extends LitElement {
           </div>
           <div class="item-creator">
             <div class="truncated">
-              <span><strong>By:&nbsp;</strong>${itemCreator}</span>
+              ${itemCreator ? html`<span>by&nbsp;${itemCreator}</span>` : ''}
```
Comment (iisa) [inline src/tiles/grid/item-tile.ts:35]:
```suggestion
              ${ this.model?.creator ? html`<span>by&nbsp;${itemCreator}</span>` : nothing }
```

nothing is usually preferred.

File: src/tiles/grid/item-tile.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export class ItemTile extends LitElement {
 
   render() {
     const itemTitle = this.model?.title || '';
-    const itemCreator = this.model?.creator || '-';
+    const itemCreator = this.model?.creator || '';
```
Comment (iisa) [inline src/tiles/grid/item-tile.ts:23]:
```suggestion
```

File: src/tiles/grid/item-tile.ts (hunk 1 above)
Comment (iisa) [reply src/tiles/grid/item-tile.ts:35]: please also add a quick unit test so we can keep coverage up. thank u!

Comment (mc2) [reply src/tiles/grid/item-tile.ts:35]: Please see: https://lit.dev/docs/api/templates/#nothing

We've been using this as the right "lit.dev" way to have the DOM cleared and empty, as @iisa suggests above.

Comment (mc2) [inline src/tiles/grid/item-tile.ts:35]: Please see: https://lit.dev/docs/api/templates/#nothing

We've been using this as the right "lit.dev" way to have the DOM cleared and empty, as @iisa suggests above.

**Review by mc2 — CHANGES_REQUESTED** [review]

**Review by mc2 — APPROVED** [review]

**Pull request merged**